build/yamls/elk-flow-collector/logstash/ipfix.yml

```diff
@@ -4070,6 +4070,24 @@
   113:
     - :string
     - :egressNetworkPolicyNamespace
+  114:
+    - :string
+    - :ingressNetworkPolicyUID
+  115:
+    - :uint8
+    - :ingressNetworkPolicyType
+  116:
+    - :int32
+    - :ingressNetworkPolicyPriority
+  117:
+    - :string
+    - :egressNetworkPolicyUID
+  118:
+    - :uint8
+    - :egressNetworkPolicyType
+  119:
+    - :int32
+    - :egressNetworkPolicyPriority
```

Comment:
while adding the policy type makes sense to me, I am not so sure about adding the UID and the priority... The UID is duplicate information once you have the type and the name. The priority is also unnecessary IMO since we can retrieve it based on the rule name. We are just growing the record size, but I don't see much benefit. If a tool needs to retrieve this information, it could be given access to the K8s API. Any opinion on this @jianjuns and @tnqn?

Reply:
For UID, yes, it means nothing for most users and it cannot be used to get a resource from K8s API. Normally it's only useful to distinguish resources having same name while existing in different time. If the records are only used for flow visibility, I would agree UID has no much value.
For priority, it seems that it's the openflow priority which is dynamically assigned on each Node, so even same policy rule will have different values when agents export them. Will it be really useful?

Reply:
Regarding UID, the use case we thought of earlier is not to show to the user but it's for integration with systems, where there is access to k8s API of the cluster and UID could be used to fetch the network policy object rather than using the name.
For priority, I was not aware that each agent computes its own priority for a given rule. If that is the case, I agree that it cannot be used. We should show policy priority or tier priority. IMO, we can remove this rule priority for now.

Reply:
> Normally it's only useful to distinguish resources having same name while existing in different time.

@tnqn Do you think it is a case that we need to handle here? If yes, it feels like we should also include the UID for the other K8s resources as well, e.g. source & destination Pods.

I feel like it's all or nothing. Either we include UIDs for all K8s resources, in which case it makes the records larger and adds complexity, or we include it for none of them. The advantage of including the UID is to make sure that we get the correct resource by name from the K8s API.

Reply:
> Regarding UID, the use case we thought of earlier is not to show to the user but it's for integration with systems, where there is access to k8s API of the cluster and UID could be used to fetch the network policy object rather than using the name.

@srikartati But UID in K8s can not be used to fetch object, the primary key in API and etcd is [namespace/]name.

@antoninbas I feel resources with same name usually mean same thing, and pods' names are usually auto-generated randomly so unlikely conflict.. Maybe we can start from not including it and consider adding it when user complains not being able to distinguish records of historial objects causes real problem.

Reply:
> @srikartati But UID in K8s can not be used to fetch object, the primary key in API and etcd is [namespace/]name.

@tnqn  I meant in systems where the K8s objects are stored in a separate inventory and given that it is organized by using UID. At the same time, we can request a different inventory organization as well.. not a concrete use-case.

I see Antonin's point that we may need UID for all objects in that case. I agree that we can remove it for now and take it up in the future if there is a strong requirement.

Reply:
Agreed. Let's remove it for now. It's easier to just add something later if we actually need it.